dev-env/tests/jest/configs/visual.setup.js

```diff
@@ -0,0 +1,4 @@
+const { toMatchImageSnapshot } = require('jest-image-snapshot');
```

Comment:
use https://github.com/wix-incubator/match-screenshot if it would be possible to get applitools key for open source